conanfile.py

```diff
@@ -68,11 +95,25 @@ def build(self):
             replace_in_file("%s/CMakeLists.txt" % self.ZIP_FOLDER_NAME, "project(libjpeg-turbo C)", "")
             
             cmake = CMake(self.settings)
-            self.run("cd %s && mkdir _build" % self.ZIP_FOLDER_NAME)
-            cd_build = "cd %s/_build" % self.ZIP_FOLDER_NAME
-            self.run('%s && cmake .. %s -DWITH_CRT_DLL=ON' % (cd_build, cmake.command_line)) # Don't change runtime, conan will take care of
-            self.run("%s && cmake --build . %s" % (cd_build, cmake.build_config))
-                
+            builddir = os.path.join(self.ZIP_FOLDER_NAME, "_build")
+
+            if os.path.exists(builddir):
```

Comment:
sometimes it did error with "Error folder is already existing!" when I used "conan install --build=SDL2_image*" twice. dunno why it happened, but I thought that may solve it and it did